doc/source/whatsnew/v1.3.0.rst

```diff
@@ -33,6 +33,80 @@ For example:
         storage_options=headers
     )
 
+.. _whatsnew_130.window_method_table:
```

Comment:
looks like you picked up another change here

Reply:
Should I merge latest?

Reply:
you should *always* merge latest every time you are pushing

Reply:
add docs for io.rst in this PR

Reply:
there is a top-level table in io.rst that needs updating as well (for the I/O read/write methods near the top)

Reply:
Added XML section and updated top-level table.

Reply:
great, this still looks like an artfiact from a merge